### PR TITLE
MDCT 371 - Update Footer

### PR DESCRIPTION
**Pull request by ntsummers1**
# Description

This "foot"-er is really going to "heel" our design woes. I'll be here all week.

Fixes [MDCT-371](https://qmacbis.atlassian.net/browse/MDCT-371)

Examples:

Desktop:
Screen Shot 2022-06-13 at 1 14 31 PM

Tablet (760px, not 768px):
Screen Shot 2022-06-13 at 1 15 11 PM

Mobile:
Screen Shot 2022-06-13 at 1 15 26 PM

## How to test

- `./dev local`
- Scroll down and check out the footer
- Adjust screen size to make sure mobile is handled properly.

## Dependencies

Please spell out any dependencies for this change -- ex -- requires an install / update / migration etc

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist

- [x] I have performed a self-review of my code
- [x] I have added thorough tests. [Testing Doc](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research)
- [x] Necessary analytics were added, or no new analytics were required
- [x] I have made corresponding changes to the documentation
- [x] I have assigned the PR to myself

## Reviewers Checklist (Two different people)

- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

## Assignee

- [ ] I have closed the PR after the review and necessary changes (squashing preferred)

### CODE REVIEW

**Review by BearHanded — DISMISSED** [review]

**Review by karla-vm — DISMISSED** [review]
The "foot"-er fix sure is a "step" in the right direction, 10/10.

File: services/ui-src/src/components/layout/Footer.js [hunk 1]
```diff
@@ -1,31 +1,55 @@
...
               <div className="chip-info ds-l-col--6">
                 <div className="medicaid-logo">
-                  <img src="/img/medicadelogo.png" alt="Medicaid.gov" />
-                </div>
-                <div className="macpro-logo">
-                  <img src="/img/MACProLogotransparent.png" alt="MACPro.gov" />
+                  <img
+                    src={cartsLogoGrey}
+                    alt="Logo for Medicaid Data Collection Tool (MDCT): CHIP Annual Report Template System (CARTS)"
+                  />
```
Comment (gmrabian) [inline services/ui-src/src/components/layout/Footer.js:18]: Nit: remember to change to ReportING here too (and in test so it still passes)

**Comment by gmrabian** [comment]
One small comment - otherwise ready to approve

**Review by gmrabian — APPROVED** [review]
Sweet feet

**Review by BearHanded — APPROVED** [review]

**Pull request merged**